meta-lmp-base/recipes-connectivity/docker-network-ref/docker-network-ref/docker-network-ref.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Create docker-network-ref docker bridge network
+After=boot-complete.target
+Before=aktualizr-lite.service
+Requires=boot-complete.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/docker-network-ref.sh
```

Comment:
I would probably just rename the script to easily show the action that is taking inside it.

create-docker-ref-network or something like that.